volumes/miovision/sql/create-table-volumes_15min_mvt.sql

```diff
@@ -1,67 +1,61 @@
 ﻿CREATE TABLE miovision_api.volumes_15min_mvt
 (
-  volume_15min_mvt_uid serial NOT NULL,
-  intersection_uid integer,
-  datetime_bin timestamp without time zone,
-  classification_uid integer,
-  leg text,
-  movement_uid integer,
-  volume numeric,
-  processed boolean,
-  CONSTRAINT volumes_15min_mvt_pkey PRIMARY KEY (volume_15min_mvt_uid),
-  CONSTRAINT volumes_15min_mvt_intersection_uid_datetime_bin_classificat_key UNIQUE (intersection_uid, datetime_bin, classification_uid, leg, movement_uid)
+    volume_15min_mvt_uid serial NOT NULL,
+    intersection_uid integer,
+    datetime_bin timestamp without time zone,
+    classification_uid integer,
+    leg text,
+    movement_uid integer,
+    volume numeric,
+    processed boolean,
+    CONSTRAINT volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey
```

Comment:
```
ERROR:  there is no unique constraint matching given keys for referenced table "volumes_15min_mvt"
```
when creating it again after dropping when transferring from mio_staging to miovision_api. We will just need to make volumes_15_min_mvt_uid unique